heron/common/src/cpp/basics/mempool.cpp

```diff
@@ -21,5 +21,5 @@
 
 // TODO(nlu): get the pool size limit from config
 MemPool<google::protobuf::Message>* __global_protobuf_pool__ =
-                                   new MemPool<google::protobuf::Message>(50 * 1024 * 1024);
+                                   new MemPool<google::protobuf::Message>(512);
```

Comment:
Got it. Could you update the TODO nlu comment above to explain why this is config-driven, but hard coded here.